### ISSUE TITLE
Catalog filters only apply to the books currently shown on a particular catalog page

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks-aldine#178]
### Description

When a user attempts to filter the list of books in a network's public catalog, the filter is only applied to those books that are currently visible on the catalog page and not other pages.

### Steps to Reproduce

1. Visit https://university.pressbooks.pub/catalog/
2. Click 'filter by license' and select 'CC-BY'
3. Watch as the list of 9 books becomes 8 books
4. Click page 2 and discover that there are 3 more CC-BY books there, none of which appeared on the list when it was filtered.

**Expected behavior:** 
Filters should apply to all the books in the catalog, irrespective of what 'page' they're on.

**Actual behavior:** 
Filters only apply to books listed on page 1, page 2, etc.

**Notes:**
We were waiting for a WordPress update to the network site directory before we could fix this. It came in WP 5.1. Notify Apurva at Rebus when implemented.

**Comment by greatislander** [comment]
>  I believe we were waiting for Wordpress update to network site directory to fix this.

Yep!

**Comment by SteelWagstaff** [comment]
@greatislander Do you happen to know if this is something that we're still waiting on or whether what we were waiting for was included with [5.1 release](https://make.wordpress.org/core/2019/01/28/multisite-support-for-site-metadata-in-5-1/)?

**Comment by greatislander** [comment]
It was included with 5.1, see: https://make.wordpress.org/core/2019/01/28/multisite-support-for-site-metadata-in-5-1/

**Comment by pb-amy** [comment]
This bug was reported on August 28th, 2020 by UW - the correspondence where this was recorded can be found on [this Zendesk ticket](https://pressbooks.zendesk.com/agent/tickets/1942).

**Comment by SteelWagstaff** [comment]
Closing in favor of #286 and #246
